src/test/java/org/svydovets/baseEntity/NotManagedPerson.java

```diff
@@ -0,0 +1,23 @@
+package org.svydovets.baseEntity;
+
+import org.svydovets.annotation.Column;
+import org.svydovets.annotation.Id;
+import org.svydovets.annotation.Table;
+
+@Table(name = "persons")
+public class NotManagedPerson {
```

Comment:
Is not understandable from name the reason of current person class, better named the same like with id. 
PersonWithoutEntityAnnotation will be more clear